index.html

```diff
@@ -1,123 +1,95 @@
-          <ul>
-            <li>
-              <a href="#"
-                ><img
-                  class="img"
-                  src="Images/twittertwitter.png"
-                  alt="Twitter logo"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img class="img" src="Images/Linkedin.png" alt="LinkedIn logo"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img
-                  class="img"
-                  src="Images/mediummedium.png"
-                  alt=" Medium logo"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img
-                  class="img"
-                  src="Images/githubgithub.png"
-                  alt="Github logo"
-              /></a>
-            </li>
-            <li>
-              <a href="#"
-                ><img
-                  class="img"
-                  src="Images/angellistangelist.png"
-                  alt="Angellist logo"
-              /></a>
-            </li>
-          </ul>
-        </div>
+    </nav>
+    <section class="front-page">
+      <div class="space"></div>
+      <h1 class="header">Hey there. I'm Cisco.</h1>
+      <h1 class="article">I'm a software developer.</h1>
+      <div>
+        <p class="p">
+          I can help you build a product, feature or website. Look through
+          some of my work and experience! If you like what you see and have a
+          project you need developed, don't hesitate to contact me.
+        </p>
+      </div>
+      <div>
         <div class="space"></div>
-        <div>
-          <img
-            class="drop-down"
-            src="Images/drop-down.png"
-            alt="drop-down button"
-          />
-        </div>
-      </section>
-    </main>
-    <section class="rest">
-      <h2 class="works" id="recent">My Recent Works</h2>
-      <div class="indicator">
-        <img id="line" src="Images/Indicator.png" alt="fancyLine"/>
       </div>
-      <div class="row">
+      <div class="images">
+        <ul>
+          <li>
+            <a href="#"><img class="img" src="Images/twittertwitter.png" alt="Twitter logo" /></a>
+          </li>
+          <li>
+            <a href="#"><img class="img" src="Images/Linkedin.png" alt="LinkedIn logo" /></a>
+          </li>
+          <li>
+            <a href="#"><img class="img" src="Images/mediummedium.png" alt=" Medium logo" /></a>
+          </li>
+          <li>
+            <a href="#"><img class="img" src="Images/githubgithub.png" alt="Github logo" /></a>
+          </li>
+          <li>
+            <a href="#"><img class="img" src="Images/angellistangelist.png" alt="Angellist logo" /></a>
+          </li>
+        </ul>
+      </div>
+      <div class="space"></div>
+      <div>
+        <img class="drop-down" src="Images/drop-down.png" alt="drop-down button" />
+      </div>
+    </section>
+  </main>
```

Comment:
I would suggest that you put this tag just before your footer. Semantically a page has a header which you have, a main section and a footer.
The main section contains several sections. 
This is so that screen readers can follow your page.

Reply:
Done